### PR TITLE
WDFTMarch2022Remote Sepid

### PR DESCRIPTION
**Pull request by Sepidehatt**
done!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,19 +3,21 @@ import './App.css';
 import { useState } from 'react'
 
 function App() {
-  const fiveContact = contactsArr.splice(0, 5);
-
+  const fiveContact = contactsArr.filter((contact,index)=> index<5);
+console.log(fiveContact)
```
Comment (TA-Remote) [inline src/App.js:7]: Alternative solution:

```
const fiveContacts = contactsArr.slice(0, 5);
```

File: src/App.js [hunk 2]
```diff
@@ -3,19 +3,21 @@ import './App.css';
...
   const addRandom = () => {
-    const randomContact = otherContacts.slice(randomIndex,randomIndex +1)
-    const addToContacts = shallowCopyContact.concat(randomContact)
-    console.log(addToContacts)
+    // const randomContact = otherContacts.splice(randomIndex,1)
+    const addToContacts = shallowCopyContact.concat(otherContacts[randomIndex])
+    // const addToContacts = shallowCopyContact.concat(randomContact)
+    // console.log(addToContacts)
     setContact(addToContacts)
   }
```
Comment (TA-Remote) [inline src/App.js:22]: To keep code more readable, I would suggest keeping all logic in separate blocks of code/separate functions.

So here, if you have a function to addRandom -> you might want to move lines 10,11  inside it.  Not to have loose lines of code

**Review by TA-Remote — COMMENTED** [review]
Amazing Sepid! Keep working hard

File: src/App.js (hunk 1 above)
Comment (Sepidehatt) [reply src/App.js:7]: i used it before,,,but i came across lots of problem,,, then i changed it,,

File: src/App.js (hunk 2 above)
Comment (Sepidehatt) [reply src/App.js:22]: cool..thank you for your suggestion

**Pull request closed without merging**